db/migrate/201607221959_add_request_help_to_user_exercise.rb

```diff
@@ -0,0 +1,5 @@
+class AddRequestHelpToUserExercise < ActiveRecord::Migration
+  def change
+    add_column :user_exercises, :request_help, :boolean, default: false
```

Comment:
If the field is renamed to `help_requested` then does a boolean make more sense?